app/helpers/ad_renewal_letters_exports_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module AdRenewalLettersExportsHelper
+  def format_email_to_name(email)
+    email.scan(/(\A.*)_(.*)@/).flatten.map(&:capitalize).join(" ")
```

Comment:
My knowledge of regex is shocking. So I used rubular to do a quick test https://rubular.com/r/EGxeUwzeNM1Qew

Based on that I think this will work fine for our seeded test email accounts, but will fail in production where they will be separated with a `.`.

Reply:
I wasn't sure about the format of the production ones, so I was expecting this :D will update the regex :)

Reply:
```suggestion
    email.scan(/(\A.*)\.(.*)@/).flatten.map(&:capitalize).join(" ")
```